src/saltext/vmware/states/esxi.py

```diff
@@ -1114,3 +1114,168 @@ def firewall_config(
                 ret["comment"] = "Configurations have successfully been changed."
 
     return ret
+
+
+def ntp_config(
+    name,
+    service_running,
+    ntp_servers=None,
+    service_policy=None,
+    service_restart=False,
+    datacenter_name=None,
+    cluster_name=None,
+    host_name=None,
+    service_instance=None,
+    profile=None,
+):
+    """
+    Set ntp configuration on matching ESXi hosts.
+
+    name
+        Name of the state.
+
+    service_running
+        Ensures the running state of the ntp daemon for the host. Boolean value where
+        ``True`` indicates that ntpd should be running and ``False`` indicates that it
+        should be stopped.
+
+    ntp_servers
+        A list of servers that should be added to the ESXi host's NTP configuration.
+
+    service_policy
+        The policy to set for the NTP service.
+        .. note::
+            When setting the service policy to ``off`` or ``on``, you *must* quote the
```

Comment:
Here's your issue -- 

```suggestion

        .. note::
        
            When setting the service policy to ``off`` or ``on``, you *must* quote the
```

I think that should fix it for you @joechainz